### PR TITLE
Melissa Brimer

### PR DESCRIPTION
**Pull request by melissabrimer**
# Checkpoint Rubric

This is the rubric that your instructor will use to grade your checkpoints. Please do not edit. 

## Checkpoint 1: Company/Organization Website following Wireframe
- [x] A CSS file is effectively linked to the HTML file in the Head section: 10 points
- [x] A link to a CSS reset file is included in the Head section of the HTML file: 10 points 
- [x] The webpage links to another page (multi-page website): 10 points
- [x] The company/orgnanization name is lined up with the navigation menu at the top of the page: 10 points
- [x] The title is centered in front of a background image: 10 points
- [x] The body images are sized and formatted according to the wire frame: 10 points
- [x] The body text is sized, positioned, and formatted to match the fire frame: 10 points
- [x] Flexbox or floats are used to accurately replicate the layout of the wireframe: 10 points
- [x] The footer consists of a background image, centered text, and 2 buttons that function as links: 10 points
- [x] The student demonstrated mastery of HTML/CSS by building a website of professional caliber: 10 points

Specific Comments:

Great job, Melissa! Outstanding website.

*Comments may also be made on the file itself using github's commenting tools.*

## Checkpoint 2: Foundry Winery Website
- [x] Responsive navbar: 20 points
- [x] Responsive site (resizes to fit desktop, tablet, and mobile views): 30 points 
- [x] Matched layout of original site: 20 points
- [x] Matched font of original site: 10 points
- [x] Properly commented code: 10 points
- [x] Properly indented code: 10 points

Specific Comments:

Well done, Melissa! Parallax... fancy!

*Comments may also be made on the file itself using github's commenting tools.*

## Checkpoint 3: Learning Team Project
- [ ] Mockup meets customer requirements: 25 points
- [ ] Website meets customer requirements: 25 points
- [ ] Learning team worked together effectively and contributed equally: 25 points
- [ ] Class presentation was engaging and effective: 25 points

Specific Comments:

*Comments may also be made on the file itself using github's commenting tools.*

## Checkpoint 4: Personal Portfolio Website
- [x] Responive: 20 points
- [x] Evidence of Effective Use of Design Principles: 20 points
- [x] Links to Resume: 20 points
- [x] Links to at least 3 Projects: 20 points
- [x] Effective Communicates Desired Message ("Hire me"): 20 points

Specific Comments:

*Comments may also be made on the file itself using github's commenting tools.*

### CODE REVIEW

**Review by msonmezates — COMMENTED** [review]
Hey Melissa, I am in group Intro-1 and need to leave a comment for an assignment so I picked yours. Hope it's fine :) 

It looks like you don't have the right indentation for your css. That's why you have an error on your last commit. I would suggest to download "Atom Beautify" from Atom > Preferences > Install Packages. Hope this clears your error after applying it.

File: 03week/blog/css/style.css [hunk 1]
```diff
@@ -1,165 +1,169 @@
...
+    -webkit-font-smoothing: antialiased;
+    min-width: 300px;
+    overflow-x: hidden;
+    overflow-y: scroll;
+    text-rendering: optimizeLegibility;
 }
 
 body {
-  color: #666;
-  font-family: "Source Sans Pro", "Helvetica", "Arial", sans-serif;
```
Comment (ethancrowley) [inline 03week/blog/css/style.css]: nice choice of font

File: 03week/blog/index.html [hunk 2]
```diff
@@ -16,163 +16,163 @@
...
-        <footer class="article-info">
-          <span>By Joe Smith</span>
-          <span>42 comments</span>
-        </footer>
-      </div>
-    </a>
+          <p class="article-content">
+            The California foursome follow their pop-punk breakthrough Never Hungover Again with the more grown-up and measured Cody.
+          </p>
+          <footer class="article-info">
```
Comment (ethancrowley) [inline 03week/blog/index.html]: good use of class

File: 03week/blog/index.html [hunk 3]
```diff
@@ -16,163 +16,163 @@
 </head>
 
 <body>
+  <header>
+    <h1>Music Blog</h1>
```
Comment (ethancrowley) [inline 03week/blog/index.html]: Header is in the right place

File: 03week/newspaper/css/style.css [hunk 4]
```diff
@@ -98,7 +98,7 @@ article img {
 
 .flex-grid-footer {
```
Comment (ethancrowley) [inline 03week/newspaper/css/style.css]: nice use of flex box

File: 03week/newspaper/css/style.css [hunk 5]
```diff
@@ -98,7 +98,7 @@ article img {
...
     display: flex;
-    background-color: darkslategray;
```
Comment (ethancrowley) [inline 03week/newspaper/css/style.css]: Love this color

File: 03week/newspaper/index.html [hunk 6]
```diff
@@ -47,19 +47,16 @@
       <article>
         <img src="img/mac.jpg" alt="Woman sitting at a computer">
```
Comment (ethancrowley) [inline 03week/newspaper/index.html]: nice linking of image

**Review by ktee1232 — COMMENTED** [review]
Great job on the Checkpoint! 

Try looking at how to add buttons in HTML :) - Kai

**Pull request closed without merging**